viewer/src/migration/types.ts

```diff
@@ -45,3 +45,12 @@ export interface AddModelOptions {
   onProgress?: (progress: OnProgressData) => void;
   onComplete?: () => void;
 }
+
+export enum WellKnownModelTypes {
```

Comment:
File3dFormat, or rename the File3dFormat to this. Supported3dFormat is another name I like more than WellKnownModelTypes :)

Reply:
I like `WellKnown[Something]` and think its a well established pattern (eg [[1]](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/enum)). However, I have no problem renaming to e.g. `SupportedModelTypes` or just `ModelTypes`